tools/cli/pkg/ers/command/status.go

```diff
@@ -0,0 +1,258 @@
+package command
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"regexp"
+
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/common/runtime"
+	mothership "github.com/kyma-project/control-plane/components/reconciler/pkg"
+	mothershipAuth "github.com/kyma-project/control-plane/components/reconciler/pkg/auth"
+	"github.com/kyma-project/control-plane/tools/cli/pkg/command"
+	"github.com/kyma-project/control-plane/tools/cli/pkg/ers"
+	"github.com/kyma-project/control-plane/tools/cli/pkg/ers/client"
+	"github.com/kyma-project/control-plane/tools/cli/pkg/logger"
+	"github.com/pkg/errors"
```

Comment:
replace "github.com/pkg/errors" with
"errors" as this is now part auf standard library and pkg/errors was archived